README.md

```diff
@@ -26,13 +26,14 @@ Before you can start the initialization of Affirm SDK, you must first set the Af
 ```java
 Affirm.initialize(new Affirm.Configuration.Builder("public key")
         .setEnvironment(Affirm.Environment.SANDBOX)
+        .setCountryCode("USA")  // Default USA
+        .setLocale(Locale.US.toString())    // Default en_US
```

Comment:
Same here `Locale.US.toLanguageTag()`

Reply:
`toLanguageTag` returns `en-US`, but what we need is `en_US` @KennethNickles